linked audiences test event draft (#6648)
segmentio/segment-docs: src/engage/audiences/linked-audiences.md

> info "Linked Audiences is in public beta"
> Linked Audiences is in public beta, and Segment is actively working on this feature. Some functionality may change before it becomes generally available.

> Linked Audiences allows you to build a warehouse first solution that powers individualized customer experiences using the relational data you've defined in your [Data Graph](/docs/unify/linked-profiles/data-graph/). 

Reviewer: ```suggestion
Linked Audiences allows you to build a warehouse-first solution that powers individualized customer experiences using the relational data you've defined in your [Data Graph](/docs/unify/linked-profiles/data-graph/). 
```

### Step 2b: Select your Destination Actions

> The [Destination Actions](/docs/connections/destinations/actions/) framework allows you to see and control how Segment sends the event data it receives from your sources to actions-based destinations. Each Action in a destination lists the event data it requires, and the event data that is optional. Segment displays available actions based on the destination you've connected to your Linked Audience. You can see details of each option and how to use it in the [Actions Destinations Catalog](/docs/connections/destinations/catalog/) documentation. 

Reviewer: ```suggestion
The [Destination Actions](/docs/connections/destinations/actions/) framework allows you to see and control how Segment sends the event data it receives from your sources to actions-based destinations. Each Action in a destination lists the event data it requires and the event data that is optional. Segment displays available Actions based on the destination you've connected to your Linked Audience. You can see details of each option and how to use it in the [Actions Destinations Catalog](/docs/connections/destinations/catalog/) documentation. 
```

After you select an action, Segment attempts to automatically configure the data fields that will be sent to the destination. You can review and adjust these settings before enabling this event.

#### Enrich event 

Select additional traits and properties to include when the event is sent.

#### Show/Hide preview 

As you're configuring your events in Linked Audiences, you can view a preview of the enriched event based on the properties you select. See what the enriched event will look like in your payload schema. 

> > important 

Author: link

> It is important to make a copy of the data from your final payload schema, as you will need this data later when you set up your destination. 

#### Map event

Only required fields that need to be filled out are displayed. All other optional & pre-filled fields are hidden - These fields are pre-filled with properties that will work by default.

## Step 3: Send a test event to your destination 

> Send a test event to see if your action-based campaign is rendering correctly. Enter the destination User id for the profile you want to use to test the event, then click the Send test event to destination button.  

Author: ensure that your destination has received the Segment custom event.

> important 
> The User ID you enter here must match a User ID you previously set up in your destination.

If the test event is not successful, adjust the User id or other properties in your destination, then test again. 

The Event content drop-down shows you a preview of what the data sent to your destination may look like. 

> ### Step 3a: Setting up your Destination Campaign 

Author: This part is where they exit the procedure to go to the new Braze/other docs.

After you select an action, Segment attempts to automatically configure the data fields that will be sent to the destination. You can review and adjust these settings before enabling this event.

#### Enrich event 

Select additional traits and properties to include when the event is sent.

> #### Show/Hide preview 

Author: this is where we'll link from the Braze docs.

Author: Casie to build this section out so it covers everything related to testing the event going into 3rd part MCM tool.

After you select an action, Segment attempts to automatically configure the data fields that will be sent to the destination. You can review and adjust these settings before enabling this event.

#### Enrich event 

Select additional traits and properties to include when the event is sent.

#### Show/Hide preview 

As you're configuring your event, you can view a preview of the enriched event based on your property selections. See what the enriched event will look like in your payload schema. 

> info 
> It is important that copy the data from your final payload schema, you will need this data later when you set up your destination. 

#### Map event
Only required fields that need to be filled out are displayed. All other optional & pre-filled fields are hidden - These fields are pre-filled with properties that will work by default.

> #### Send test event to destination 

Author: As a prerequisite to sending a test event, you must have defined some things in your destination already. 

Define a user prior to triggering campaigns based on the custom event.

After you select an action, Segment attempts to automatically configure the data fields that will be sent to the destination. You can review and adjust these settings before enabling this event.

#### Enrich event 

Select additional traits and properties to include when the event is sent.

#### Show/Hide preview 

As you're enriching your events in Linked Audiences, you should view a preview of the event payload schema based on the properties you select. It might look like the following:

> ![{Preview your payload data}](/docs/engage/images/linked_audience_payload.png)

Reviewer: ```suggestion
![A screenshot of the Add activation page, where you can review your payload data.](/docs/engage/images/linked_audience_payload.png)
```

**Important:** It is important to make a copy of the data from your final payload schema; you will need this data later when you set up your destination. 

#### Map event

> Only required fields that need to be filled out are displayed. All other optional & pre-filled fields are hidden 

Reviewer: ```suggestion
Only required fields are displayed. All optional & pre-filled fields are hidden.
```

These fields are pre-filled with properties that will work by default.

## Step 3: Send a test event to your destination

> Send a test event to ensure that everything is connected properly and your destination is receiving the event. 

Reviewer: ```suggestion
Send a test event to ensure that everything is connected properly and your destination receives the event. 
```

> Enter the destination User id for the profile you want to use to test the event, then click the **Send test event to destination** button.  

Reviewer: ```suggestion
Enter the destination User id for the profile you want to use to test the event, then click **Send test event to destination**.  
```

> The Event content drop-down shows you a preview of what the data sent to your destination may look like. 

Reviewer: ```suggestion
The Event content drop-down shows you a preview of what the data sent to your destination might look like. 
```

> ### Step 3a: Configure your multi-channel marketing campaign 

Author: When I get a separate PR up for this topic (Braze), I'll link to it.

> If you're using a multi-channel marketing tool, set up your email campaign in your destination before contiuing to Step 4. 

Reviewer: ```suggestion
If you're using a multi-channel marketing tool, set up your email campaign before contiuing to Step 4. 
```

## Step 4: Enable your Linked Audience

> After building your Linked Audience, and choose **Save and Enable**, you'll be redirected to the Audience Overview page. Segment automatically disables your audience so that it does not start computing until you're ready. To enable your audience:
> 
> Select the **Enabled** toggle, then select **Enable audience**.
> 
> You can trigger a compute for your audience if you want to send events to your destination without waiting for the next scheduled compute run. To do so, select **Compute Now**. This triggers a compute for the audience (where the audience conditions run on your data warehouse) and sends events downstream.

Reviewer: ```suggestion
After building your Linked Audience, choose **Save and Enable**. You'll be redirected to the Audience Overview page, where you can view the audience you created. Segment automatically disables your audience so that it doesn't start computing until you're ready. A compute is when Segment runs the audience conditions on your data warehouse and sends events downstream. 

To enable your audience:
Select the **Enabled** toggle, then select **Enable audience**.

You can trigger a compute for your audience if you want to send events to your destination without waiting for the next scheduled compute run. To do so, select **Compute Now**. This triggers a compute for the audience and sends events downstream.
```
